test_ui.py

```diff
@@ -13,16 +13,16 @@ def test_ui(browser):
     page = browser.new_page()
 
     # Mock the fetch request to /extract-text
-    page.route("**/extract-text?url=http%3A%2F%2Fexample.com", lambda route: route.fulfill(
+    page.route("**/extract-text?url=https%3A%2F%2Fthinkingofbermuda.com", lambda route: route.fulfill(
```

Comment:
Stop using mocked data here. https://thinkingofbermuda.com/ is a real website that should have images.